src/cmd_line.rs

```diff
@@ -197,6 +201,11 @@ pub fn handle_command_line_arguments(args: Vec<String>) -> Result<(), String> {
             .map(|v| v.to_owned())
             .or_else(|| std::env::var("NEOVIDE_X11_WM_CLASS").ok())
             .unwrap_or_else(|| "neovide".to_owned()),
+        // Command-line arguments with multiple files
```

Comment:
Rather than throwing the comment all over the place, it could be more useful to rename the variable to something like `command_line_files` or `target_files` instead.

Reply:
Hey, that looks good to me, I'll update with this change now :D